projects/sandbox/analyze/analyze.py

```diff
@@ -173,6 +181,11 @@ def build_background(
             y, t = seg.load("out")
```

Comment:
Might be worthwhile to just have `load_segments` return `y, t`, since it doesn't seem like we need the segment object for anything once the data's been loaded